monai/apps/auto3dseg/algorithms/templates/dints/configs/hyper_parameters_search.yaml

```diff
@@ -0,0 +1,55 @@
+---
+# hyper-parameters
+amp: true
+bundle_root: null
+arch_path: "$@bundle_root + '/arch_ram' + str(@ram_cost_factor) + '_fold' + str(@fold)"
```

Comment:
As above, the fold may be removed or replaced by a fix string

Reply:
the intention to do so is to enable saving model folders of 5-fold under the same bundle dir instead of creating five different copies.